docs/core/contributing.md

```diff
@@ -5,7 +5,7 @@ title: Contribution Guide
 
 Our goal is to make contributing to the Libra project easy and transparent.
 
-<blockquote class="block_note">
+<blockquote className="block_note">
```

Comment:
All .md changes (except demo page) were to add the className to quiet the warnings. We should most likely remove these.